packages/appstore/src/components/main-title-bar/index.tsx

```diff
@@ -21,9 +24,20 @@ const MainTitleBar = () => {
         setActiveIndex(selected_region === 'Non-EU' ? 0 : 1);
     }, [selected_region]);
 
+    const { is_wallet_enabled } = useFeatureFlags();
+
+    // TODO: Remove this when we have BE API ready
+    removeAllNotificationMessages(true);
+
+    React.useEffect(() => {
+        filterNotificationMessages();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
     return (
         <React.Fragment>
             <DesktopWrapper>
+                {is_wallet_enabled && <WalletsBanner />}
                 <div className='main-title-bar'>
                     <div className='main-title-bar__right'>
                         <Text size='m' weight='bold' color='prominent'>
```

Comment:
since this `{is_wallet_enabled && <WalletsBanner />}` should be on top of the component for both mobile and desktop, could we not do something like this?

```suggestion
        <React.Fragment>
            {is_wallet_enabled && <WalletsBanner />}
            <DesktopWrapper>
                <div className='main-title-bar'>
                    <div className='main-title-bar__right'>
                        <Text size='m' weight='bold' color='prominent'>
```

Reply:
Updated.